src/infill/ZigzagConnectorProcessor.h

```diff
@@ -43,20 +43,6 @@ namespace cura
  * - with disconnected endpieces
  * - with connected endpieces
  * 
- * Each of these has a base class for which ZigzagConnectorProcessor is an ancestor.
- * The inheritance structure is as such:
- *                                                                      ZigzagConnectorProcessor
- *                                                                           /             \                                    .
- *                                                                          /               \                                   .
- *                                                  ActualZigzagConnectorProcessor      NoZigZagConnectorProcessor
- *                                                         /                \             for lines infill                                      .
- *                                                        /                  \                                                  .
- *                          ZigzagConnectorProcessorEndPieces   ZigzagConnectorProcessorNoEndPieces
- *                                     /            \                 for zigzag infill (without end pieces)                                                          .
- *                                    /              \                                                                          .
- * ZigzagConnectorProcessorConnectedEndPieces     ZigzagConnectorProcessorDisconnectedEndPieces
- * for zigzag support with normal endpieces          for zigzag support with disconnected endpieces for more easy removability
- * 
```

Comment:
This sketch is to complain the complex relationships of those classes. Since now we have cleaned it up, I added a small comment on the NoZigzagConnector (in 81e93783)